docker-compose-cd.yml

```diff
@@ -0,0 +1,88 @@
+version: "3"
+
+services:
+  ####################### Client Definition #######################
+  client:
+    image: client:${GIT_COMMIT:-latest}
+    container_name: client
+    build:
+      context: client
+    command: ["npm", "run", "start"]
+    environment:
+      HOST: 0.0.0.0
+      PORT: 3000
+    volumes:
+      - ./client/src/:/opt/client/src/
+      - "/app/node_modules"
+    ports:
+      - 80:3000
+    networks:
+      - default
+
+  ####################### Database Definition #######################
+  database:
+    image: database:${GIT_COMMIT:-latest}
+    container_name: database
+    build:
+      context: database
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER:-admin}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-password}
+      - POSTGRES_DB=dm_library
```

Comment:
is this "library" naming specific to the last SWU, too?

Reply:
oh yeah, guessing dm = Digital Marketplace here

Reply:
Yes! good catch, updating